examples/acs2/maze/acs2_in_maze.py

```diff
@@ -21,29 +22,36 @@ def maze_metrics(population, environment):
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-e", "--environment", default="BMaze4-v0")
+    parser.add_argument("--epsilon", default=1.0, type=float)
+    parser.add_argument("--ga", action="store_true")
```

Comment:
How does `action="store_true"` works? Shouldn't we also add explicit type here (like `type=bool`)?

Reply:
store_true means that if that switch is passed on the command line, args.ga variable will be True; otherwise it will be False

Reply:
Explicit type specification is not necessary in store_true case, according to examples from argparse documentation